### ISSUE TITLE
Taints on Windows nodes

### ISSUE THREAD

**Issue by vakkursw** [aws-samples/aws-eks-accelerator-for-terraform#270]
#### Please describe your quesiton here
When creating a self-managed windows node, i cannot taint the nodes using kubelet_extra_args = "--register-with-taints=kubernetes.io/os=windows:NoSchedule"

  # SELF-MANAGED NODE GROUP with Windows support
  enable_windows_support = true

  self_managed_node_groups = {
    ng_od_windows = {
      node_group_name    = "selfwindows"
      launch_template_os = "windows"
      instance_type      = "m5n.large"
      subnet_ids         = module.aws_vpc.private_subnets
      min_size           = "1"
      kubelet_extra_args = "--register-with-taints=kubernetes.io/os=windows:NoSchedule"
    }
![2022-02-18_14-45-38](https://user-images.githubusercontent.com/95715828/154758878-7db8c13b-6f7c-4a9d-87dc-73d88df05141.png)

**Comment by awsitcloudpro** [comment]
Apparently recent changes removed users' ability to pass launch template / user data parameters such as `kubelet_extra_args`. @ricardosouzamorais can you please address this?

@vakkursw until this is resolved, you can use the older release v3.2.1 of this module to pass `kubelet_extra_args`, e.g.

```
module "aws-eks-accelerator-for-terraform" {
  source = "git@github.com:aws-samples/aws-eks-accelerator-for-terraform.git?ref=v3.2.1"
  ...
  self_managed_node_groups = {
    ng_od_windows = {
      ...
      kubelet_extra_args = "--register-with-taints kubernetes.io/os=windows:NoSchedule"
      ...
    }
  }
}
```

**Comment by ricardosouzamorais** [comment]
Sure.

**Comment by ricardosouzamorais** [comment]
> Apparently recent changes removed users' ability to pass launch template / user data parameters such as `kubelet_extra_args`. @ricardosouzamorais can you please address this?
> 
> @vakkursw until this is resolved, you can use the older release v3.2.1 of this module to pass `kubelet_extra_args`, e.g.
> 
> ```
> module "aws-eks-accelerator-for-terraform" {
>   source = "git@github.com:aws-samples/aws-eks-accelerator-for-terraform.git?ref=v3.2.1"
>   ...
>   self_managed_node_groups = {
>     ng_od_windows = {
>       ...
>       kubelet_extra_args = "--register-with-taints kubernetes.io/os=windows:NoSchedule"
>       ...
>     }
>   }
> }
> ```

Sure. I will dive deep into this.
